### PR TITLE
first commit

### PR DESCRIPTION
**Pull request by jacopo-polito**
approvaci sta pull request

### CODE REVIEW

**Review by jacopo-polito — COMMENTED** [review]
Is it necessary to have .vsccode folder in the repo?

File: Jacopo/intro.c [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#include <stdio.h>
+
+int main (){
+    printf("Hello world!");
+
+    return 0;
+}
```
Comment (jacopo-polito) [inline Jacopo/intro.c:7]: way better now

**Comment by leonardo-bracci** [comment]
> Is it necessary to have .vsccode folder in the repo?

I think the .vscode folder is generated automatically for .c files (and .js?) to specify what compiler to use, the c language version, the path to the #include headers and shit like this. 

I think you can get rid of it, but you have to specify all that in the command line when you compile the program

**Review by leonardo-bracci — APPROVED** [review]
Looks all good to me, we can merge to main branch

**Comment by jacopo-polito** [comment]
Ok

**Comment by leonardo-bracci** [comment]
approvaci sta pull request

**Review by jacopo-polito — COMMENTED** [review]
Review changes v2

**Pull request merged**